js/pagination.js

```diff
@@ -0,0 +1,72 @@
+var currentPage=1;
```

Comment:
defining variables like this will most likely cause conflicts. Please use  a function to wrap